lib/show_n_tell/vote.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ShowNTell.Vote do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+
+  schema "votes" do
+    field :value, :integer
+
+    belongs_to :user, ShowNTell.Vote
+    belongs_to :talk, ShowNTell.Vote
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(vote, attrs) do
+    vote
+    |> cast(attrs, [:value])
```

Comment:
This helps allot. Thank you!
I was trying to run your line (10) with errors.
I found I had to prefix the model name with ShowNTell, like so.
`%ShowNTell.Vote{} |> ShowNTell.Vote.changeset(%{user_id: 1, talk_id: 1, value: 1}) |> ShowNTell.Repo.insert!`

Reply:
@cgregs32 

> I found I had to prefix the model name with ShowNTell, like so.

Yes, you can also do an alias to avoid typing that out

`alias ShowNTell.Vote`

or if you have multiple things you want to alias `alias ShowNTell.{Vote, User, Talk}